src/intervals/intervals.jl

```diff
@@ -38,16 +38,46 @@ eltype{T<:Real}(x::Interval{T}) = T
 
 
 ## Include files
-include("special_intervals.jl")
-include("printing.jl")
-include("rationalize.jl")
-include("parameters.jl")
-include("rounding.jl")
-include("macro_definitions.jl")
-include("conversion_promotion.jl")
+include("special.jl")
+include("macros.jl")
+include("conversion.jl")
 include("arithmetic.jl")
 include("precision.jl")
```

Comment:
The file "precision.jl" contains, among others, the `IntervalParameters` type and other rounding functions. Though it seems to create no problems for the tests, for sanity I think it is better to include this file *before* any actual use of parameters or rounding stuff. In particular, include it before "arithmetic.jl".